app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -192,4 +191,18 @@ public void shouldNotAccessView() {
 
         verifyZeroInteractions(eventDetailView);
     }
+
+    @Test
+    public void shouldHideProgressbarCorrectly() {
+        when(eventRepository.getAttendees(id, false))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        when(eventRepository.getEvent(id, false))
+            .thenReturn(Observable.just(event));
+
+        eventDetailPresenter.start();
+
+        assertTrue(eventDetailPresenter.getProgress() >= 2);
+        verify(eventDetailView).showProgressBar(false);
```

Comment:
This is only verifying that the progress bar was hidden. And the shown test is in two separate tests. I  was talking about moving the show and hide test in single one and remove the show test from those two